examples/structured_data/classification_with_tfdf.py

```diff
@@ -146,23 +160,14 @@
 """
 
 
-def prepare_sample(features, target, weight):
-    for feature_name in features:
-        if feature_name in CATEGORICAL_FEATURES_WITH_VOCABULARY:
-            if features[feature_name].dtype != tf.dtypes.string:
-                # Convert categorical feature values to string.
-                features[feature_name] = tf.strings.as_string(features[feature_name])
-    return features, target, weight
-
-
 def run_experiment(model, train_data, test_data, num_epochs=1, batch_size=None):
 
     train_dataset = tfdf.keras.pd_dataframe_to_tf_dataset(
         train_data, label=TARGET_COLUMN_NAME, weight=WEIGHT_COLUMN_NAME
-    ).map(prepare_sample, num_parallel_calls=tf.data.AUTOTUNE)
+    )
     test_dataset = tfdf.keras.pd_dataframe_to_tf_dataset(
         test_data, label=TARGET_COLUMN_NAME, weight=WEIGHT_COLUMN_NAME
-    ).map(prepare_sample, num_parallel_calls=tf.data.AUTOTUNE)
+    )
 
     model.fit(train_dataset, epochs=num_epochs, batch_size=batch_size)
     _, accuracy = model.evaluate(test_dataset, verbose=0)
```

Comment:
**Cannot comment in this block anymore.

>> Sort

I think tf.sort and unique are done in memory.
It would be interesting to understand and document what append in this processing layer when used on a large dataset. For example, will the entire dataset be loaded in memory?